src/awesome/property.rs

```diff
@@ -0,0 +1,63 @@
+use rlua::{self, Lua, Value, ToLua, FromLua};
+use super::class::PropF;
+
+pub struct Property<'lua> {
```

Comment:
Add documentation explaining this isn't userdata and that it's just a Rust representation of a table from Lua.